src/Domain.LinnApps/Reports/ChangeStatusReportService.cs

```diff
@@ -0,0 +1,411 @@
+﻿using Linn.Common.Reporting.Layouts;
```

Comment:
usings should be inside namespaces - there's quite a few of these in this PR so maybe your stylecop isn't functioning?